pyproject.toml

```diff
@@ -120,16 +125,24 @@ runtimes = "elyra.metadata.schemasproviders:RuntimesSchemas"
 runtimes-images = "elyra.metadata.schemasproviders:RuntimeImagesSchemas"
 code-snippets = "elyra.metadata.schemasproviders:CodeSnippetsSchemas"
 component-catalogs = "elyra.metadata.schemasproviders:ComponentCatalogsSchemas"
+# Airflow is not part of ODH distribution
+# airflow-provider-package-catalog-schema = "elyra.pipeline.airflow.provider_package_catalog_connector.airflow_provider_package_schema_provider:AirflowProviderPackageSchemasProvider"
```

Comment:
seems like all these commented out, is this intentional ?

Reply:
Yes. These commented out lines were removed on this fork so I'm bringing them back, but commented out so that Airflow-releated stuff is not included in the whl file. Should we keep them uncommented?

Reply:
yes for now, lets keep them uncommented, for keep the whl file light. 
as we are syncing with upstream, we can iron out more of these, to keep the difference as minimal as possible. 
we can check what would the size be if include or not include airflow, and time of elyra load on runtime, if any impact.

Reply:
Sounds great, I agree! I'll look into it in a separate PR.